scripts/make/qemu.mk

```diff
@@ -26,9 +26,14 @@ qemu_args-aarch64 := \
 
 qemu_args-y := -m 128M -smp $(SMP) $(qemu_args-$(ARCH))
 
-qemu_args-$(BLK) += \
-  -device virtio-blk-$(vdev-suffix),drive=disk0 \
-  -drive id=disk0,if=none,format=raw,file=$(DISK_IMG)
+ifeq ($(findstring driver-nvme,$(FEATURES)), driver-nvme)
```

Comment:
I am not sure if this can be configured through env var `$(FEATURES)`, if the user only enable this feature in `Cargo.toml`, it may not exist in `$(FEATURES)` @equation314

Reply:
@ZR233 You could add a new make argument `BLK_DEV` like `NET_DEV`.